docker/Dockerfile-build

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
```

Comment:
Sorry for jumping in at a random point, but I was curious about why we are using `16.04` here (as opposed to say a newer LTS like `18.04`)? Is it because production runs on ubuntu `16.04`?

Reply:
It's probably fine to move to the latest LTS release. This PR was merely about moving the external dockerfiles into this repository from https://github.com/valhalla/docker